### PR TITLE
LL-4662 Tables style update

### PR DESCRIPTION
**Pull request by LFBarreto**
(Tables): updated styles for remaining not updated tables (tokens, votes, delegation, rewards)

localhost_8080_webpack_index html_theme=null
localhost_8080_webpack_index html_theme=null_
localhost_8080_webpack_index html_theme=null__
localhost_8080_webpack_index html_theme=null___

### Type

UI Polish
### Context

LL-4662
### Parts of the app affected / Test plan

Mainly tron, algorand, cosmos, polkadot accounts page UI, on tokens, delegation and rewards sections.

### CODE REVIEW

**Review by valpinkman — DISMISSED** [review]

**Comment by valpinkman** [comment]
good but you need to update the screenshots :)

File: src/renderer/families/polkadot/components/CollapsibleList.js
Comment (rjarasson-ledger) [inline src/renderer/families/polkadot/components/CollapsibleList.js:66]: why passing a containerStyle (with { margin: 0})  and not removing margins entirely here ?
Or remove the card to only have the collapsible effect ?

I think this component should become generic (and merged with the TokenAccounts show more).

Comment (LFBarreto) [reply src/renderer/families/polkadot/components/CollapsibleList.js]: yeah sorry i though this component was used elsewhere. indeed a simple box is better. As for a generic way to do this i can do it but i think this one wont benefit from it since it has a very specific way of splitting between visible and hidden items :shrug:

Comment (rjarasson-ledger) [reply src/renderer/families/polkadot/components/CollapsibleList.js]: @LFBarreto Yup, it is specific, but maybe a good arrangement of collapsible component (only logic), with the atoms & molecules from Design System (table/show more footer...) could allow use to do reusable work.

Not a priority though, there are more important topic. Thanks for the fix.
I have a PR (and some more after) that could lead to conflicts, i'll try to test yours asap - could be great to be merged before we add more conflicts on polkadot.

File: src/renderer/families/polkadot/Nomination/index.js
Comment (rjarasson-ledger) [inline src/renderer/families/polkadot/Nomination/index.js:373]: It breaks on Polkadot.

Nominations and Unbondings should be 2 separate Tables.
I might separate the 2 sections into 2 components (included in the AccountBodyHeader.js) after your PR, since we might have a 3rd section for Rewards that would come soon.

This component is too long.


image

**Review by rjarasson-ledger — CHANGES_REQUESTED** [review]

File: src/renderer/families/polkadot/Nomination/index.js
Comment (rjarasson-ledger) [inline src/renderer/families/polkadot/Nomination/index.js:378]: typo here, we are loosing the tooltip because of it.

"tolltip" ->"tooltip"

**Review by rjarasson-ledger — CHANGES_REQUESTED** [review]
Looks good on Polkadot, Cosmos and Algorand on light theme

But on dark & dusk themes, i noticed this rendering - buttons (inverted) are light and does not quite fit the general aspect of LLD - might be an implementation bug of Buttons that should be addressed in another ticket and should not block this PR ?

Happens on other coins too.

image

**Review by rjarasson-ledger — APPROVED** [review]

File: src/renderer/families/polkadot/Nomination/index.js
Comment (rjarasson-ledger) [inline src/renderer/families/polkadot/Nomination/index.js:280]: Just noticed a non-related related bug.
This `mr` is not supposed to be here, nominate is not aligned correctly.

If you don't want to fix this in this PR, i can plan to fix this while doing a split of this file after your PR is merged.

**Review by rjarasson-ledger — DISMISSED** [review]

**Review by valpinkman — DISMISSED** [review]

**Comment by valpinkman** [comment]
@LFBarreto needs rebase

**Comment by valpinkman** [comment]
@LFBarreto snapshot update needed

**Comment by valpinkman** [comment]
@LFBarreto with the merge of Annoucement feature, all screenshots need to be updated (also a rebase is necessary I guess)

**Review by valpinkman — DISMISSED** [review]

**Comment by valpinkman** [comment]
Test are flaky, not blocker for merging

**Review by rjarasson-ledger — APPROVED** [review]

**Review by nabil-brn — CHANGES_REQUESTED** [review]
Buttons are white in dark mode, the primary style is what we used until now.

image

---
914eb2d03 (HEAD -> LL-4662, luiz/LL-4662) Merge branch 'develop' of github.com:LedgerHQ/ledger-live-desktop 
into LL-4662

**Review by nabil-brn — CHANGES_REQUESTED** [review]
Withdraw button on polkadot account page looks not disabled visually but it is.

image

---
a42cabe4e (HEAD -> LL-4662, luiz/LL-4662) update screenshots

**Review by nabil-brn — APPROVED** [review]

**Review by valpinkman — APPROVED** [review]

**Pull request merged**
